tests/oauth2/test_oauth2_admin.py

```diff
@@ -0,0 +1,113 @@
+from collections.abc import Set
+from unittest.mock import patch
+from regtech_api_commons.oauth2.config import KeycloakSettings
+from regtech_api_commons.oauth2.oauth2_admin import OAuth2Admin
+import jose.jwt
+
+
+kc_settings = KeycloakSettings()
+oauth2_admin = OAuth2Admin(kc_settings)
+
+
+def test_get_claims():
+    token = "Test Token"
+
+    with patch.object(oauth2_admin, "_get_keys", return_value="secret"):
```

Comment:
instead of mocking the `_get_keys`, we should mock the `requests.get`, then you can do that `mock.assert_called_with...` check

Reply:
@lchen-2101 Thanks! Working on it.